yajsapi/executor/index.ts

```diff
@@ -129,6 +135,7 @@ export class Executor {
   private _expires;
   private _budget_amount;
   private _budget_allocations: Allocation[];
+  private _rejecting_providers: Set<string>;
```

Comment:
In the Python version the corresponding set is stored in `AgreementsPool` which is created for each `Executor.submit()` call. Here one set is used through possibly many calls to `Executor.submit()`.

I guess it's fine, maybe even better than in the Python version?